# Use system paths
diff --git a/Cura/util/pluginInfo.py b/Cura/util/pluginInfo.py
index 7290c8b..52fa14b 100644
--- a/Cura/util/pluginInfo.py
+++ b/Cura/util/pluginInfo.py
@@ -80,14 +80,7 @@ def setPostProcessPluginConfig(config):
 	profile.putProfileSetting('plugin_config', pickle.dumps(config))
 
 def getPluginBasePaths():
-	ret = []
-	if platform.system() != "Windows":
-		ret.append(os.path.expanduser('~/.cura/plugins/'))
-	if platform.system() == "Darwin" and hasattr(sys, 'frozen'):
-		ret.append(os.path.normpath(os.path.join(resources.resourceBasePath, "plugins")))
-	else:
-		ret.append(os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', 'plugins')))
-	return ret
+	return [os.path.expanduser('~/.cura/plugins/'), '/usr/share/cura/plugins']
 
 def getPluginList(pluginType):
 	global _pluginList
diff --git a/Cura/util/resources.py b/Cura/util/resources.py
index e63230b..99bb5ce 100644
--- a/Cura/util/resources.py
+++ b/Cura/util/resources.py
@@ -28,7 +28,7 @@ if sys.platform.startswith('darwin'):
 	else:
 		resourceBasePath = os.path.join(os.path.dirname(__file__), "../../resources")
 else:
-	resourceBasePath = os.path.join(os.path.dirname(__file__), "../../resources")
+	resourceBasePath = "/usr/share/cura"
 
 def getPathForResource(dir, subdir, resource_name):
 	assert os.path.isdir(dir), "{p} is not a directory".format(p=dir)
