diff -ur Cura-2.5.0.orig/tests/TestMachineAction.py Cura-2.5.0/tests/TestMachineAction.py
--- Cura-2.5.0.orig/tests/TestMachineAction.py	2017-04-21 13:30:29.000000000 +0200
+++ Cura-2.5.0/tests/TestMachineAction.py	2017-05-17 00:02:31.273106246 +0200
@@ -30,19 +30,19 @@
         machine_manager.addMachineAction(test_action)
 
     # Check that the machine has no supported actions yet.
-    assert machine_manager.getSupportedActions(test_machine) == set()
+    assert machine_manager.getSupportedActions(test_machine) == []
 
     # Check if adding a supported action works.
     machine_manager.addSupportedAction(test_machine, "test_action")
-    assert machine_manager.getSupportedActions(test_machine) == {test_action}
+    assert machine_manager.getSupportedActions(test_machine) == [test_action]
 
     # Check that adding a unknown action doesn't change anything.
     machine_manager.addSupportedAction(test_machine, "key_that_doesnt_exist")
-    assert machine_manager.getSupportedActions(test_machine) == {test_action}
+    assert machine_manager.getSupportedActions(test_machine) == [test_action]
 
     # Check if adding multiple supported actions works.
     machine_manager.addSupportedAction(test_machine, "test_action_2")
-    assert machine_manager.getSupportedActions(test_machine) == {test_action, test_action_2}
+    assert machine_manager.getSupportedActions(test_machine) == [test_action, test_action_2]
 
     # Check that the machine has no required actions yet.
     assert machine_manager.getRequiredActions(test_machine) == set()
@@ -53,11 +53,11 @@
 
     ## Check if adding single required action works
     machine_manager.addRequiredAction(test_machine, "test_action")
-    assert machine_manager.getRequiredActions(test_machine) == {test_action}
+    assert machine_manager.getRequiredActions(test_machine) == [test_action]
 
     # Check if adding multiple required actions works.
     machine_manager.addRequiredAction(test_machine, "test_action_2")
-    assert machine_manager.getRequiredActions(test_machine) == {test_action, test_action_2}
+    assert machine_manager.getRequiredActions(test_machine) == [test_action, test_action_2]
 
     # Ensure that firstStart actions are empty by default.
     assert machine_manager.getFirstStartActions(test_machine) == []
